test/cql-pytest/nodetool.py

```diff
@@ -83,3 +83,14 @@ def refreshsizeestimates(cql):
         pass
     else:
         run_nodetool(cql, "refreshsizeestimates")
+
+def bloom_filter_false_positives(cql, table):
+    ks, cf = table.split('.')
+    if has_rest_api(cql):
+        response = requests.get(f"{rest_api_url(cql)}/column_family/metrics/bloom_filter_false_positives/{ks}:{cf}")
+        return response.json()
+    else:
+        # Nodetool has no method to obtain this information. This is part of `nodetool tablestats`
+        pass
```

Comment:
This doesn't fit nodetool.py very well, because it's not a nodetool command.

I think what we need is a more generic get_metric() command, which takes the metric name as a parameter - we don't need a separate function for each of the metrics. I guess maybe you can keep it in nodetool.py, or just put it in any other file, and it can still import nodetool.py and use `nodetool.has_rest_api()` and `nodetool.rest_api_url()`.

Also, we need to do something in the "else"... Perhaps a pytest.skip() because this test can't run if the REST API is not available.